BalancedTree/BalancedTree.js

```diff
@@ -0,0 +1,60 @@
+class TreeNode {
+    constructor(val) {
+      this.val = val;
+      this.left = null;
+      this.right = null;
+    }
+  }
+  
+  function isBalanced(root) {
+    // Subtree Calculation
+    function getHeight(node) { 
+      if (node === null) {
```

Comment:
Because `null` is already falsey in JS, you don't need to explicitly check against `null` here. It's actually better not to, because you're more likely to come across `undefined` than `null` in JS, and `undefined === null` evaluates to `false`.
```suggestion
      if (!node) {
```